File: OCR-APP/ui/src/app/classification-response.ts
export class ClassificationResponse {
    constructor(
        public requestOwner: string,
        public requestCategory: string,
        public totalKeywords: number,
        public matchedKeywords: number,
        public confidence: number,
        public docType: string,
        public requestStatus: string,
        public dateCreated: Date,
        public classificationResult: string
    ){}
}

File: OCR-APP/backend/src/main/java/com/classisfication/DocClassification/entity/ClassificationResponse.java
package com.classisfication.DocClassification.entity;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class ClassificationResponse {

	private String requestOwner;
	
	private String requestCategory;
	
	private long totalKeywords;
	
	private long matchedKeywords;
	
	private long confidence;
	
	private String classificationResult;
	
}
